content/about.html

```diff
@@ -1,7 +1,7 @@
 +++
 title = "Meet GYC West"
 description = "Mission statement, purpose, and more about GYC West"
-date = "2022-01-07"
+date = "2025-04-18"
```

Comment:
The date stamp is really supposed to be updated when someone last made an intentional change to the page.